.travis.yml

```diff
@@ -2,9 +2,12 @@ language: ruby
 cache: bundler
 rvm:
   - 1.9.3
```

Comment:
Also, codeclimate/codeclimate runs on 1.9.3. We might want to keep it around until we're on 2.x bandwagon.